src/organizations/serializers/organization.serializer.ts

```diff
@@ -8,7 +9,7 @@ export const deserializeOrganization = (
   name: organization.name,
   allowProfilesOutsideOrganization:
     organization.allow_profiles_outside_organization,
-  domains: organization.domains,
+  domains: organization.domains.map(deserializeOrganizationDomain),
```

Comment:
previously the domains were not deserialized and as a result had fields like `verification_token` and `verification_strategy` instead of `verificationToken` and `verificationStrategy`. this is why i updated the test above.